edge-hub/core/test/Microsoft.Azure.Devices.Edge.Hub.Core.Test/routing/RoutingTest.cs

```diff
@@ -504,6 +526,11 @@ class IoTHub
             public bool HasReceivedTwinChangeNotification() => this.ReceivedMessages.Any(
                 m =>
                     m.SystemProperties[SystemProperties.MessageType] == Constants.TwinChangeNotificationMessageType);
+
+            public bool HasReceivedMessageFromModule(string deviceId, string moduleId) => this.ReceivedMessages.Any(
```

Comment:
Instead of this, I would say update the HasReceivedTwinChangeNotification to check if it came from edgeHub.